flow360/component/simulation/operating_condition.py

```diff
@@ -113,8 +114,8 @@ def dynamic_viscosity(self) -> ViscosityType.Positive:
         """Computes dynamic viscosity."""
         # pylint: disable=fixme
         # TODO: implement
-        # return self.material.dynamic_viscosity(self.temperature)
-        return 1.825e-5 * u.Pa * u.s
+        return self.material.dynamic_viscosity_from_temperature(self.temperature)
```

Comment:
Have you wrote the viscosity + temperature --> Sutherland's coefficient helper function?